boards/ti/sk_am62/board.cmake

```
@@ -8,3 +8,8 @@ if(CONFIG_SOC_AM6234_M4)
   board_runner_args(openocd "--no-init" "--no-halt" "--no-targets" "--gdb-client-port=3339")
   include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
 endif()
+
+if(CONFIG_SOC_AM6234_A53)
+  board_runner_args(openocd "--no-init" "--no-halt" "--no-targets" "--gdb-client-port=3339")
```

Each core has a different port:
```
Info : starting gdb server for am625.cpu.sysctrl on 3333
Info : Listening on port 3333 for gdb connections
Info : starting gdb server for am625.cpu.a53.0 on 3334
Info : Listening on port 3334 for gdb connections
Info : starting gdb server for am625.cpu.a53.1 on 3335
Info : Listening on port 3335 for gdb connections
Info : starting gdb server for am625.cpu.a53.2 on 3336
Info : Listening on port 3336 for gdb connections
Info : starting gdb server for am625.cpu.a53.3 on 3337
Info : Listening on port 3337 for gdb connections
Info : starting gdb server for am625.cpu.main0_r5.0 on 3338
Info : Listening on port 3338 for gdb connections
Info : starting gdb server for am625.cpu.gp_mcu on 3339
Info : Listening on port 3339 for gdb connections
```


```suggestion
  board_runner_args(openocd "--no-init" "--no-halt" "--no-targets" "--gdb-client-port=3334")
```